setuptools/command/egg_info.py

```diff
@@ -239,7 +322,151 @@ def check_broken_egg_info(self):
 
 
 class FileList(_FileList):
-    """File list that accepts only existing, platform-independent paths"""
+    # Implementations of the various MANIFEST.in commands
```

Comment:
I kept it for backwards compatibility. Packages or plugins may depend on calling `FileList.include_pattern`, the old way of adding packages, and I did not want to break these.